convert2rhel/actions/__init__.py

```diff
@@ -540,17 +641,28 @@ def run_actions():
     results = system_checks.run()
 
     # Format results as a dictionary:
-    # {"$Action_id": {"status": int,
-    #                 "error_id": "$error_id",
-    #                 "message": "" or "$message"},
+    # {
+    #     "$Action_id": {
+    #         "messages": [
+    #             "level": int,
+    #             "id": "$id",
+    #             "message": "" or "$message"
+    #         ]
+    #     },
+    #     "result": {
+    #         "level": int,
+    #         "id": "$id",
+    #         "message": "" or "$message"
+    #     },
     # }
```

Comment:
```suggestion
    #     "$Action_id": {
    #         "messages": [
    #             "level": int,
    #             "id": "$id",
    #             "message": "" or "$message"
    #         ]
    #         "result": {
    #             "level": int,
    #             "id": "$id",
    #             "message": "" or "$message"
    #         },
    #     },
    # }
```

The structure is a tiny bit different than this.  `result` and `messages` are both cildren of `$Action_id`.